cmd/kubeapps-apis/Dockerfile

```diff
@@ -8,12 +8,27 @@ WORKDIR /go/src/github.com/vmware-tanzu/kubeapps
 COPY go.mod go.sum ./
 ARG VERSION="devel"
 
+# If true, run golangci-lint to detect issues
+ARG lint
+
+# https://github.com/bufbuild/buf/releases/
 ARG BUF_VERSION="1.25.1"
+
+# https://github.com/golangci/golangci-lint/releases
 ARG GOLANGCILINT_VERSION="1.53.3"
 
+# https://github.com/grpc-ecosystem/grpc-health-probe/releases/
+ARG GRPC_HEALTH_PROBE_VERSION="0.4.19"
+
 # Install lint tools
+RUN if [ ! -z "$lint" ]; then \
+        go install github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANGCILINT_VERSION; \
+    fi
+
 RUN curl -sSL "https://github.com/bufbuild/buf/releases/download/v$BUF_VERSION/buf-Linux-x86_64" -o "/tmp/buf" && chmod +x "/tmp/buf"
-RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANGCILINT_VERSION
+
+# TODO: Remove and instead use built-in gRPC container probes once we're supporting >= 1.24 only. https://kubernetes.io/blog/2022/05/13/grpc-probes-now-in-beta/
```

Comment:
Just noticed GKE_STABLE_VERSION is now 1.25, so we can remove this and update the chart too (not here).

Reply:
Yeah, we could potentially do it, but it can have some unintended side effects if we sync our chart with Bitnami's. AFAIK, there is still some k8s 1.23 testing in place :(